src/pages/iou/ReceiptSelector/NavigationAwareCamera.js

```diff
@@ -21,10 +28,56 @@ const defaultProps = {
     torchOn: false,
 };
 
+function useTabNavigatorFocus({cameraTabIndex, isInTabNavigator}) {
+    // Get navigation to get initial isFocused value (only needed once during init!)
+    const isPageFocused = useIsFocused();
+    const [isTabFocused, setIsTabFocused] = useState(false);
+
+    // Retrieve the animation value from the tab navigator, which ranges from 0 to the total number of pages displayed.
+    // Even a minimal scroll towards the camera page (e.g., a value of 0.001 at start) should activate the camera for immediate responsiveness.
+
+    // STOP!!!!!!! This is not a pattern to be followed! We are conditionally rendering this hook becase when used in the edit flow we'll never be inside a tab navigator.
```

Comment:
This comment is nice, but it sure makes me think that this is built wrong. It sounds like there should be two different components.

Reply:
I didn't request this change because there's already separate GH for this.
- https://github.com/Expensify/App/issues/29527

Reply:
OK, thank you for making me aware of it.